internal/cmd/group.go

```diff
@@ -210,12 +212,33 @@ func destroyGroup(client *turso.Client, name string) error {
 func groupsTable(groups []turso.Group) [][]string {
 	var data [][]string
 	for _, group := range groups {
-		row := []string{group.Name, formatLocations(group.Locations, group.Primary), group.Version, formatBool(group.Archived)}
+		row := []string{group.Name, formatLocations(group.Locations, group.Primary), group.Version, aggregateGroupStatus(group)}
 		data = append(data, row)
 	}
 	return data
 }
 
+func aggregateGroupStatus(group turso.Group) string {
+	status := "Healthy 🟢"
```

Comment:
WDYT about dropping the emojis? I don't think they add too much next to the words and they're also unpadded.